packages/@storefront/flux-capacitor/src/core/requests/utils.ts

```diff
@@ -182,6 +182,11 @@ namespace RequestHelpers {
 
   export const chain = <T>(...fns: Array<(...obj: any[]) => T>): T =>
     fns.reduce((final, fn) => fn(final) || final, <T>{});
+
+  export const attachSessionId = <T>(state: Store.State): ((r: T) => T&{ sessionId: Store.SessionId }) => {
+    const sessionId = state.session.sessionId;
```

Comment:
Unless you're going to set a fallback, just inline this.

ie.

```
...{ sessionId: state.session.sessionId }...
```

Reply:
Done.